src/KF.Records.UseCases/Records/GetAllRecords/GetAllRecordsQueryHandler.cs

```diff
@@ -36,12 +40,7 @@ public GetAllRecordsQueryHandler(IReadWriteDbContext readWriteDbContext, ILogger
     public async Task<IList<GetRecordDto>> Handle(GetAllRecordsQuery request, CancellationToken cancellationToken)
     {
         logger.LogInformation("Request for all records");
-        var records = await readWriteDbContext.Records.Select(record => new GetRecordDto()
-        {
-            Id = record.Id,
-            Description = record.Description,
-            Tags = record.Tags.ToList()
-        }).ToListAsync(cancellationToken);
+        var records = await readWriteDbContext.Records.ProjectTo<GetRecordDto>(mapper.ConfigurationProvider).ToListAsync(cancellationToken);
```

Comment:
Лучше разделить эту конструкция по строкам. Иначе не читается.
```suggestion
        var records = await readWriteDbContext.Records
            .ProjectTo<GetRecordDto>(mapper.ConfigurationProvider)
            .ToListAsync(cancellationToken);
```